src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
+  caption="Distribution of the number of `<img>` elements per page.",
   description="Bar chart showing the number of `<img>` elements per page per percentile (10, 25, 50, 75, and 90). The median desktop page features 21 `<img>` elements and the median mobile page features 19 `<<img>` tags..",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=923860709&format=interactive",
   sheets_gid="1317352222",
   sql_file="pages_markup_by_device_and_percentile.sql"
-  )
-}}
+) }}
 
-The median web page is missing 2.99% `alt` attributes on desktop and 2.44% of `alt` attributes on mobile.
+{# TODO(authors): Add your interpretation of these results. What do you hope readers get from it? Also consider linking to the Accessibility chapter here if that ties in. #}
+The median web page is missing 2.99% of `alt` attributes on desktop and 2.44% of `alt` attributes on mobile.
 
+{# TODO(analysts): Does this caption sound accurate? I wanted to clarify what it was a percent of. #}
 {{ figure_markup(
   image="seo-percentage-of-missing-img-alt-attribute.png",
   alt="Percent of missing image alt attributes",
-  caption="Percent of missing image `alt` attributes",
+  caption="Distribution of the percent of `<img>` elements missing image `alt` attributes per page.",
   description="Bar chart showing the percent of missing `alt` attributes per percentile (10, 25, 50, 75, and 90). The median web page is missing 2.99% `alt` attributes on desktop and 2.44% of `alt` attributes on mobile.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=862590664&format=interactive",
   sheets_gid="1317352222",
   sql_file="pages_markup_by_device_and_percentile.sql"
   )
 }}
 
-It was also found that only 53.86% of desktop and 51.22% of mobile home pages featured image `alt` attributes. Although this might seem low, it is an improvement from [last year](../2019/seo#image-alt-tags) when it was found that only 46.71% of mobile pages used `alt` attributes on all of their images.
-`
+{# TODO(authors, analysts): I don't think the interpretation of these stats was correct, so I edited out anything that I didn't think was accurate. Please double check my edit and feel free to expand on it. Note that this is incompatible with last year's stat because that was a Lighthouse audit checking that all images have alt attributes, while this data was calculated directly from the markup. There's something off if 40+% of pages have perfect alt coverage, but this chart has less than perfect coverage at the 75th percentile. Either coverage got much _worse_ this year or they're measuring different things. For example Lighthouse may be more lenient about which images should have alt attributes. Do you have those audit results based on 2020 data here for better comparison? #}
```

Comment:
⚠️ @aleyda @Tiggerito

Reply:
@rviscomi I'm copying @ipullrank who took care of the image section... can you please take a look to see if you're ok with the changes that have been made? It might be worthy to highlight the difference in the data source to make sure it also explains the difference from last year. cc @Tiggerito

Reply:
I'm not sure how Lighthouse classifies a blank alt attribute, that code is well hidden. I think they class as good. Definitely a different metric as that just reports if a page is perfect or not.